app/views/wiki/edit.html.erb

```diff
@@ -21,6 +21,13 @@
       <li><a href="/wiki/authoring-help#Advanced+formatting"><%= t('wiki.edit.advanced_formatting') %></a></li>
     </ul>
   </div>
+  <hr />
+  <div class="sidebar-nav hidden-sm">
+    <ul class="nav nav-list">
+      <li class="nav-header"><h4>Templates</h4></li>
```

Comment:
****

### Templates

Use these to start a page from [one of these templates](https://publiclab.org/wiki/wiki-templates):

* [Method template lite]()
* [Method template]()

****
As commented by @jywarren here https://github.com/publiclab/plots2/issues/2469#issue-303990987 , I guess template still needs to be updated to resemble above view. Thanks.